staubli_resources/urdf/common_colours.xacro

```diff
@@ -2,9 +2,15 @@
 <robot xmlns:xacro="http://ros.org/wiki/xacro">
 
   <!-- RAL 1028 -->
-  <xacro:property name="colour_staubli_yellow" value="1.0 0.608 0 1" />
+  <xacro:property name="melon_yellow" value="1.0 0.608 0.0 1.0" />
```

Comment:
Also: we cannot remove the `colour_staubli_yellow` (yet), as we do not know if/where it has been used. We should add a `colour_ral_melon_yellow` and an alias called `colour_staubli_yellow` (which has `value="${colour_ral_melon_yellow}"`).